mappings/net/minecraft/client/world/ClientEntityManager.mapping

```diff
@@ -0,0 +1,30 @@
+CLASS net/minecraft/class_5582 net/minecraft/client/world/ClientEntityManager
+	FIELD field_27279 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_27280 entityLoader Lnet/minecraft/class_5576;
+	FIELD field_27281 entities Lnet/minecraft/class_5570;
+	FIELD field_27282 sectionedEntities Lnet/minecraft/class_5573;
+	FIELD field_27283 tickingChunkSections Lit/unimi/dsi/fastutil/longs/LongSet;
+	FIELD field_27284 entityStore Lnet/minecraft/class_5577;
+	METHOD <init> (Ljava/lang/Class;Lnet/minecraft/class_5576;)V
+		ARG 1 entityBaseClass
```

Comment:
In the sectioned entity storage, this is just `entityClass`. Do they represent different things, or would it be fair to name them the same?